clients/admin-ui/src/features/system/history/SystemHistoryTable.tsx

```diff
@@ -23,7 +23,10 @@ import { SystemHistoryResponse } from "~/types/api";
 import { SystemResponse } from "~/types/api/models/SystemResponse";
 
 import {
+  alignCustomFields,
```

Comment:
ah, CI says this function doesn't exist :)

Reply:
Fixed!